deps/source_package/debian/compat

```diff
@@ -0,0 +1 @@
+7
```

Comment:
You can set this to 10

https://www.debian.org/doc/manuals/maint-guide/dother.en.html#compat